docs/controls/Bubble.md

```diff
@@ -0,0 +1,342 @@
+# Bubble
+
+**Note: Bubble will be deprecated in a future release. Use [BubbleSingleDataset](./BubbleSingleDataset.md) or [BubbleMultipleDataset](./BubbleMultipleDataset.md).**
```

Comment:
If it's gonna be deprecated, does it need to be added here?

Reply:
We don't know when the next major release will be, so we added this note here for current consumers.